apps/system/test/unit/accessibility_test.js

```diff
@@ -25,6 +26,26 @@ suite('system/Accessibility', function() {
     options: { pattern: [40], isKey: false }
   };
 
+  var vcChangeHintsEnabledDetails = {
+      eventType: 'vc-change',
+      data: [{string: 'link'}],
+      options: {
+        hints: [{string: 'link-hint'}]
+      }
+  };
+
+  var screenChangeScreenDisabledEvent = {
+    type: 'screenchange',
+    detail: {
+      screenEnabled: false
+    }
+  };
+
+  var aDataPlusHints = [
```

Comment:
I think you don't need to use it, just pass vcChangeHintsEnabledDetails.options.hints on line 368.